### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marclundgren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ library to compile the native extensions.
 ### OSX
 
     xcode-select --install
+    rvm install 1.9
```
Comment (grobie) [inline README.md]: This adds a dependency on `rvm`, or is that part of OSX now? Isn't there a ruby provided by OSX anymore?

Comment (marclundgren) [inline README.md]: well, OSX ships with ruby 2.0.0 installed. In order to get 1.9, `rvm` is the way to go. Similar to `nvm`, I would argue that `rvm` is a quintessential dependency to ruby version management.

Comment (georgi) [inline README.md]: doesn't it work with 2.0?

Comment (marclundgren) [inline README.md]: it wasn't work for me on 2.0

Comment (grobie) [inline README.md]: Well, I use rbenv ;) I'm fine with the rvm dependency if 1.9 is necessary, but then we would need to explain how to install rvm.

**Comment by marclundgren** [comment]
> I use rbenv ;)

Oh I didn't know about this. In that case, since rvm is the not the de facto standard, i'll close this PR

**Pull request closed without merging**